server/testdriver/core/src/main/java/org/infinispan/server/test/core/AbstractServerConfigBuilder.java

```diff
@@ -0,0 +1,83 @@
+package org.infinispan.server.test.core;
+
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+
+import java.util.Properties;
+
+/**
+ * Common code for JUnit 4 and Junit 5 Extension
+ *
+ * @param <T> type of builder
+ * @author Katia Aresti
+ * @since 11
+ */
+public abstract class AbstractServerConfigBuilder<T> {
```

Comment:
I suggest `AbstractTestServerBuilder`, because it creates a `TestServer` instance. Although `TestServer` isn't just one server either, it's a cluster, so maybe `AbstractTestClusterBuilder`?

Reply:
I don't agree. The Rule is called "InfinispanServerRuleBuilder" not "InfinispanClusterRuleBuilder". This builder emerged up to reuse the same type of build for the InfinispanServerExtension and InfinispanServerRule.
The InfinispanServer is either a cluster or a standalone server.